### DISCUSSION TITLE
Brainstorm for new features/triggers for m3/commands

### DISCUSSION THREAD

**Discussion by joep26020** [winstxnhdw/lc-hax#335] [Ideas]
Let's try to brainstorm ideas for new features I think that would make it easier for Winston. 

1.  Ability to set seed for ship (maybe you have a favorite map or there's a map with the mask on it)
2. When doing /mask it automatically grabs and hold the mask for you
3. a /mute voice command to mute other people's microphone for all users
4. Spoof host (-probably not possible, I know. Just thinking about it.)
5. Touch TP - whenever the right click is pressed the user will teleport to the location looked at in the crosshair
6. Ability to carry an unlimited amount of items
7. ability to activate functions of monsters (-I know, needs work. Just thinking about it)
8. Teleport enemies a singular entity to a location rather than all at once
9. infinite teleporter/inverse use (no wait time)
10. crash server command (I've found that spawning 400+ mask usually lags the server to make it unplayable.)
11. instant push - when colliding with another player, or a player collides with you, you experience no knockback force from them. This could also be used in conjunction with the follow command but  increase the intensity on the follow command to make you basically fling a player. 
12. No name, change in game name
13. make it so nameless name doesn't occur anymore, and also whenever somebody clicks on your name in the player list, the placeholder name when the message: kick [name] comes up it doesn't say a different name but stays consistent with normal name. 
14. attach constant rejoin to steam ID- this would allow you to join server after host has left and restarted
15. talk through somebody elses in game mic 

Please add more and give me some suggestions on mine! And thank you Winston for this project! We appreciate you greatly!

**Comment by winstxnhdw** [comment]
1. Host-only.

2. I'll look into it.

3. Not possible.

4. Not possible.

5. Low-priority but I'll look into it.

6. Not worth the effort and time. This is probably enough work to be an entire mod on its own.

7. In the works.

8. Sure. Maybe a `/mob <enemy>`.

9. You can already do this with interact at a distance.

10. No.

11. Interesting idea.

12. Low priority. Will look into it.

13. Low priority.

14. It already does this.

15. Low priority.

**Comment by Dexteriority** [comment]
we really need that number 2 xD

**Comment by joep26020** [comment]
> 1. Host-only.
> 2. I'll look into it.
> 3. Not possible.
> 4. Not possible.
> 5. Low-priority but I'll look into it.
> 6. Not worth the effort and time. This is probably enough work to be an entire mod on its own.
> 7. In the works.
> 8. Sure. Maybe a `/mob <enemy>`.
> 9. You can already do this with interact at a distance.
> 10. No.
> 11. Interesting idea.
> 12. Low priority. Will look into it.
> 13. Low priority.
> 14. It already does this.
> 15. Low priority.

thanks for the reply lol I like the answer to number 10 😂! I have to ask though, for number 14, for me it only tries to join again one time, not multiple times.

**Comment by winstxnhdw** [comment]
The following are now in latest: 2, 7, 9, 14
